src/Components/SideDrawer/SideDrawer.jsx

```diff
@@ -43,6 +43,7 @@ export default function SideDrawer() {
     if (noteHash !== undefined) {
       const extractedCommentId = noteHash.split(':')[1]
       setSelectedNoteId(Number(extractedCommentId))
+      console.log('isDrawerOpen: ', isDrawerOpen)
```

Comment:
remove.

Reply:
@OlegMoshkovich, thank you, i removed all `console.log`.